src/issue/screens/issue.screen.js

```diff
@@ -214,14 +220,25 @@ class Issue extends Component {
     />;
 
   render() {
-    const { issue, comments, isPendingComments, navigation } = this.props;
+    const {
+      issue,
+      comments,
+      isPendingComments,
+      isPendingIssue,
+      navigation,
+    } = this.props;
 
     return (
       <ViewContainer>
-        {isPendingComments &&
-          <LoadingContainer animating={isPendingComments} center />}
+        {(isPendingComments || isPendingIssue) &&
+          <LoadingContainer
+            animating={isPendingComments || isPendingIssue}
+            center
+            text={'Retrieving issue'}
```

Comment:
Not sure if we want the text here (yet, at least), it would be the only place in the app besides searches where we have loading text AFAIK. If we add it here I'd also want to add it to users, code and repositories. It might fit better in another PR.